rasa/core/actions/action.py

```diff
@@ -359,6 +354,37 @@ def intent_name_from_action(action_name: Text) -> Text:
         """Resolve the name of the intent from the action name."""
         return action_name.split(UTTER_PREFIX)[1]
 
+    def get_full_retrieval_name(
+        self, tracker: "DialogueStateTracker"
+    ) -> Optional[Text]:
+        """Returns full retrieval name for the action.
+
+        Extracts retrieval intent from response selector and
+        returns complete action utterance name.
```

Comment:
Can we please follow the Google spec for docstrings? I think not following it exactly is only okay for one liners